### PR TITLE
fix: apply theme when select series

### PR DESCRIPTION
**Pull request by jung-han**
### Please check if the PR fulfills these requirements
- [ ] It's submitted to right branch according to our branching model
- [ ] It's right issue type on title
- [ ] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

* asis
image

* tobe
image

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by jungeun-cho — APPROVED** [review]
리뷰완료했습니다.

File: apps/chart/src/component/areaSeries.ts [hunk 1]
```diff
@@ -695,10 +687,8 @@ export default class AreaSeries extends Component {
       throw new Error(message.SELECT_SERIES_API_INDEX_ERROR);
     }
 
-    this.eventBus.emit('renderSelectedSeries', {
-      models: [model],
-      name: this.name,
-    });
+    const models = this.getResponderSeriesWithTheme([model], 'select');
```
Comment (js87zz) [inline apps/chart/src/component/areaSeries.ts:690]: 요런건(`select`) 테마 관련 상수 만들어서 사용하는건 어떠신가요? 반복되는 곳이 많을 것 같아서요

**Review by js87zz — APPROVED** [review]
고생하셨습니다! 리뷰 완료입니다.

File: apps/chart/src/component/areaSeries.ts (hunk 1 above)
Comment (jung-han) [reply apps/chart/src/component/areaSeries.ts:690]: 일단은 타입으로 좁혀놨는데 해당케이스가 점점 많아지면 상수로 분리하겠습니다!

**Review by dotaitch — COMMENTED** [review]
늦었지만... 리뷰 완료합니다

**Pull request merged**